src/services/document-services.ts

```diff
@@ -121,4 +121,57 @@ export default class DocumentService {
       document.body.removeChild(link)
     })
   }
+
+  /**
+   * Fetches existing document records from the Document Record Service (DRS)
+   * using the provided document class and query parameters.
+   *
+   * @param documentClass - The document class to search under (e.g., 'CORP', 'FIRM').
+   * @param queryParams - Key-value pairs for query parameters (e.g., consumerDocumentId).
+   * @returns A promise resolving to the Axios response or the error response.
+   */
+  static async searchDocument (params: DocumentRequestIF): Promise<AxiosResponse> {
```

Comment:
Small update to the params described in the doc bloc

Reply:
Thank you, sir! I’ve just addressed it.